bindings/nodejs/lib/types/wallet/transaction.ts

```diff
@@ -36,6 +36,7 @@ export class Transaction {
     incoming!: boolean;
     /** Note that can be set when sending a transaction and is only stored locally */
     note?: string;
+    /** TODO. */
```

Comment:
```suggestion
    /** 
    * Outputs that are used as input in the transaction.
    * May not be all, because some may have already been deleted from the node.
    */
```